README.md

```diff
@@ -1,61 +1,86 @@
 # Development Stages via git branch naming convention
 
+## Purpose
+
+The `staged.dependencies` package simplifies the development process for developing a set of
+inter-dependent R packages. In each repository of the set of R packages you are co-developing you should
+specify a `staged_dependencies.yaml` file containing *upstream* (i.e. those packages this repo depends on) and
+*downstream* (i.e. those packages which depend on your current repo's package) 
+dependency packages within your development set of packages.
```

Comment:
thanks, very nice. It might also mention the word internal vs. external dependencies (language used in `bazel`), where `staged.dependencies` is for defining internal dependencies.